Backpack/src/main/res/layout/bpk_dialog.xml

```diff
@@ -1,42 +1,83 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout
-  xmlns:android="http://schemas.android.com/apk/res/android"
+<net.skyscanner.backpack.dialog.internal.DialogInsetsLayout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
   android:layout_height="match_parent"
-  android:background="@android:color/transparent" >
+  tools:background="@color/bpkGray500">
 
-  <androidx.appcompat.widget.LinearLayoutCompat
-    android:id="@+id/dialog_content_layout"
+  <net.skyscanner.backpack.dialog.internal.DialogWindowLayout
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:layout_alignParentTop="true"
-    android:layout_centerHorizontal="true"
-    android:background="@color/bpkWhite"
-    android:orientation="vertical" >
-
-    <net.skyscanner.backpack.text.BpkText
-      android:id="@+id/dialog_title"
-      android:layout_width="match_parent"
-      android:layout_height="wrap_content"
-      android:textAlignment="center"
-      app:textStyle="xl"
-      app:weight="emphasized" />
-
-    <net.skyscanner.backpack.text.BpkText
-      android:id="@+id/dialog_description"
-      android:layout_marginTop="@dimen/bpkSpacingMd"
-      android:layout_marginBottom="@dimen/bpkSpacingLg"
-      android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.appcompat.widget.LinearLayoutCompat
+      android:layout_width="0dp"
       android:layout_height="wrap_content"
-      android:textAlignment="center"
-      app:textStyle="lg" />
-  </androidx.appcompat.widget.LinearLayoutCompat>
-
-  <net.skyscanner.backpack.dialog.BpkDialogIcon
-    android:id="@+id/dialog_icon"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
-    android:layout_alignParentTop="true"
-    android:layout_centerHorizontal="true" />
-</RelativeLayout>
+      android:orientation="vertical"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      app:layout_constraintWidth_max="@dimen/bpk_dialog_preferred_width">
```

Comment:
The name of this dimen is a bit misleading, it's called `preferred_width` but it's used as max width, preferred to me means this will be the ideal one but can grow bigger. 

Can we change it to `bpk_dialog_max_width`

Reply:
Done